.github/workflows/build-and-publish-next.yml

```diff
@@ -1,37 +0,0 @@
-name: Publish LFNext images to Docker Hub on-demand
```

Comment:
Did you intend to delete this file? I didn't see any mention of deleting this workflow in the PR - is it still useful?

Reply:
It's not useful anymore